src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -14,78 +12,39 @@ export type Props = {
   data?: BreadcrumbItemData[];
 };
 
-const MAX_LIMIT_BREADCRUMB_LENGTH = 4;
-const MAX_ITEMS_TO_SHOW_AFTER_COLLAPSE = 2;
-const MAX_ITEMS_TO_SHOW_BEFORE_COLLAPSE = 1;
-
 const Breadcrumb: React.FC<Props> = props => {
   const { children, data = [] } = props;
-  const passDataToRouterLink = ({ to, label }: BreadcrumbItemData) => (
-    <Link css={breadcrumbLinkStyles()} key={to} to={to}>
-      {label}
-    </Link>
-  );
-  const childrenCollection = React.Children.toArray(children);
-  const dataItems = isEmpty(data) ? childrenCollection : data.map(passDataToRouterLink);
 
   const isLastItem = (dataItems: React.ReactNode[], itemIndex: number) =>
     itemIndex === dataItems.length - 1;
 
-  //Checks if an item is in collapsed area and if should break the breadcrumb items with a collapsed view
-  const shouldCollapse = React.useCallback(
-    (item: React.ReactNode, itemIndex: number) =>
-      item &&
-      dataItems.length > MAX_LIMIT_BREADCRUMB_LENGTH &&
-      itemIndex >= MAX_ITEMS_TO_SHOW_BEFORE_COLLAPSE &&
-      itemIndex < dataItems.length - MAX_ITEMS_TO_SHOW_AFTER_COLLAPSE,
-    [dataItems]
-  );
+  const passDataToRouterLink = (dataItem: BreadcrumbItemData, index: number) => {
```

Comment:
`passDataToRouterLink` ( as well as `isLastItem` ) don't need to access to scope variables and can be moved outside the component ( or even to a utils file )